src/opnsense/service/templates/OPNsense/Unbound/core/domainoverrides.conf

```diff
@@ -0,0 +1,17 @@
+{% if not helpers.empty('OPNsense.unboundplus.domains.domain') %}

```

Comment:
personally would prefer leaving the "unboundplus" namespace for default "unbound" if we are going to migrate anyway

Reply:
Wouldn't this require a larger change beyond the scope of this PR? Just my 2 cents.

Reply:
I'm not sure if model renames are a good idea for general model migration behaviour reasons (@AdSchellevis would know better) but except this the change would entail changing a few configuration template lines and that isn't unreasonable to do so during the migration taking place here anyway. M1_0_0.php already contains the code to be used for this. It takes the contents from the old config location and moves it to the new model.

Reply:
Agreed. Impact doesn't seem to be that high. I'd also like to know what @AdSchellevis thinks about model renaming.

Reply:
@swhite2 @fichtner I don't mind that much about renaming the model, but to limit impact it might be better to test and merge this first and add another model version explicitly for the move. race conditions may lead to unintended side affects.

Reply:
Only concern is that templates need to be changed after adding more of them to migrate but the end result is still I can hope for 😊